charts/ecr-proxy/values.yaml

```diff
@@ -57,16 +57,6 @@ env: []
 #    value: https://someAccountId.dkr.ecr.someRegion.amazonaws.com
 #  - name: AWS_REGION
 #    value: someRegion
-#  - name: AWS_ACCESS_KEY_ID
```

Comment:
please leave these as they are still valid for users choosing to go with secret & env-vars